.rubocop.yml

```diff
@@ -3,8 +3,23 @@ require: rubocop-rspec
 AllCops:
   TargetRubyVersion: 3.0
 
+RSpec/ExampleLength:
+  Max: 18
+
+RSpec/MultipleExpectations:
+  Enabled: false
+
+RSpec/MultipleMemoizedHelpers:
+  Enabled: false
+
+RSpec/NestedGroups:
+  Max: 4
+
```

Comment:
These are fairly loose settings. **MultipleExpectations** and **MultipleMemoizedHelpers** in particular are very helpful in keeping specs clean. Multiple assertions in a test is regarded as a smell in TDD generally, and too many memoized helpers is a pretty clear sign that you need to refactor your code. Consider removing these settings from here then run `rubocop --auto-gen-config` to generate a **.rubocop_todo.yml**. This gives you a set of goals you can work toward over time.